### ISSUE TITLE
stop go lag spike behaver 12/11/18

### ISSUE THREAD

**Issue by tenkun2017** [DarkPacks/SevTech-Ages#3509]
Bug Report:
 After loading and getting in the game, Their is a brief time of it running smooth then when moving or looking around it starts a stop and go action (lag spike behaver) except I am sitting around 60 fps at the time it happens

change/improvement- smooth game play.

all current settings are default.

actions I have tried taking to solve
install current version of java as of 12/11/18
uninstall/reinstall sev tech

 Steps to Reproduce are unknown.

* Modpack Version: 3.0.8
* Java Version: current as of 12/11/18
* Launcher Used: twich
* rivin half algorithm 
* Memory Allocated: currently off. however when off/on it still happens
* Server/LAN/Single Player: Single Player
* Optifine Installed: no and no clue how.
* Shaders Enabled: if its default then yes

 *World Information: 3.0.8

* Modpack Version world created in: 3.0.8

hp omen i7 16gb ram NVidia 1060 

forge- 14.23.4.2707
here is a gif showing it starting and stopping with f3 on in the game to show what my pc/game is doing
when playing  
![ezgif com-optimize](https://user-images.githubusercontent.com/45801014/49830562-6347fb80-fd46-11e8-9b3d-3f2bdb70b859.gif)

Note: my programing knowledge is copy and past programing. I am a regular game player who really, really want's to play this pack. I just recently got a pc that can run this pack and i am still having issues running it. 

Thank you for your time and consideration.

**Comment by XarenHypr** [comment]
That lag spike is cause by java's garbage collection function.  If you look at the memory allocation/usage in the upper righthand corner, you'll see it 'lag' when it hits 100%, then after the spike it will be lower again.  By the looks of things, you have about 3GB set up for the pack, correct?

**Comment by tenkun2017** [comment]
unintentionally maybe... I just down loaded it and ran it as it was, so if that was the default setting how would I change it to run with more? I did try to allocate it by clicking on the button under better fps but nothing seamed to change.

**Comment by tenkun2017** [comment]
thank you for your help. I learned via your implication that their is a setting some where so I have been digging around in the settings pulling my hair out looking for answers. I found nothing in Minecraft but, I did find it in twitch.

I had no clue that it had an option in the settings for memory allocation.(I don't stream, only mod)

If any one runs in to the same issue I did go to settings select Minecraft scroll down to java settings and adjust from their.
